src/epfl-people/inspector.js

```diff
@@ -43,7 +43,7 @@ export default class InspectorControlsPeople extends Component {
                 className="field-with-no-margin-bottom"
                 onChange={ structure => setAttributes( { structure } ) }
             />
-            <p>{ __('You can enter the name of a custom structure.', 'epfl') } <a target="_blank" href="https://www.epfl.ch/campus/services/ressources-informatiques/publier-sur-le-web-epfl/people/page-106273-fr-html/">{ __('More information', 'epfl') }</a>. <a target="_blank" href="https://people.epfl.ch/cgi-bin/upldtmpl">{ __('List of existing structures', 'epfl') }</a></p>
+            <p>{ __('You can enter the name of a custom structure.', 'epfl') } <a target="_blank" href="https://www.epfl.ch/campus/services/website/ressources-informatiques/publier-sur-le-web-epfl/people/page-106273-fr-html/">{ __('More information', 'epfl') }</a>. <a target="_blank" href="https://people.epfl.ch/cgi-bin/upldtmpl">{ __('List of existing structures', 'epfl') }</a></p>
```

Comment:
L'URL est incorrecte, faudrait regarder avec Hicham quelle est la nouvelle. J'ai un peu cherché et j'ai rien trouvé...

Reply:
Je crois que c'est https://www.epfl.ch/campus/services/website/web-services/help-for-people-epfl-ch/tools-for-webmasters/, en tout cas d'après archive.org

Reply:
https://web.archive.org/web/20190310095624/https://www.epfl.ch/campus/services/ressources-informatiques/publier-sur-le-web-epfl/people/page-106273-fr-html/